.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Question
+    url: https://neovim.discourse.group/
+    about: Please ask and answer questions about Neovim on Discourse.
+  - name: Chat
+    url: https://gitter.im/neovim/neovim
+    about: Join Gitter to chat with fellow Neovim users.
```

Comment:
Maybe link to matrix instead of gitter, as those two merged now, we should advertise matrix more.